modules/custom/moj_resources/src/Plugin/rest/resource/TermResource.php

```diff
@@ -35,13 +35,6 @@
  *          type="integer",
  *          description="ID of term to return",
  *      ),
- *      @SWG\Parameter(
```

Comment:
The language parameter is not actually used in the API.

Reply:
If it's not used you're right to remove it, it feels like if we could be in a position to generate the Swagger documentation again it would definitely help for it to be up to date 👍

Reply:
Yeah, I noticed the translate function wasn't being run so there was no need for it.